source/_posts/Improving-your-Azure-Search-performance.md

```diff
@@ -5,24 +5,24 @@ tags:
 
 # Azure Search
 <figure><img src="/images/search/azure-search-diagram.svg" /><figcaption style="font-style: italic; text-align: center;"><a href="https://docs.microsoft.com/bs-cyrl-ba/azure/search/search-what-is-azure-search">Azure Cognitive Search</a></figcaption></figure>
-Azure Cognitive Search can index various data sources and make them searchable by text queries. It lets you connect multiple data sources such as Cosmos DB collections, SQL databases or even documents such as PDFs. The indexing engine then indexes the content based on the index definition that has been configured. In most cases the indexation takes place on databases such as SQL databases or Cosmos DB, enabling users to index, for instance, their e-commerce platform or knowledge base.
+Azure Cognitive Search can index various data sources and make them searchable by text queries. It lets you connect multiple data sources such as Cosmos DB collections, SQL databases or even documents for example PDFs. The engine then indexes the content based on the index definition that has been configured. In most cases the indexation takes place on databases, enabling users to index, for instance, their e-commerce platform or knowledge base.
 
-Once the content has been indexed, one can query the index with a text query and additionally add facets, filters and sorting. This query will be evaluated against the index and the system will return the highest scored documents. This blog post is about unraveling the scoring system, and to help you improve your search performance. Performance matters because the user experience of the search is primarily determined by the relevance of the returned documents. In projects with Azure Cognitive Search I have come across way too many questions like: How did this document score this high? And why does it also find these documents? This blog post will help you answer such questions.
+Once the content has been indexed, one can query the index with a text query and additional facets, filters or sorting. The query will be evaluated against the index and the system will return the highest scored documents. This blog post is about unraveling the scoring system, and to help you improve your search performance. Performance matters because the user experience of the search is primarily determined by the relevance of the returned documents. In my projects with Azure Cognitive Search I came across way too many questions like: How did this document score this high? And why does it also find these documents? This blog post will help answer such questions for you.
```

Comment:
opzich had deze change niet gehoeven (additioanl) maar tijdens lezen voelde dit beter

and -> or, je kan volgens mij toch bepaalde facets filters en sorting, met and ziet het er eerder uit alsof je het allemaal tegelijkertijd moest